cli/cobra.go

```diff
@@ -29,7 +29,7 @@ func SetupRootCommand(rootCmd *cobra.Command) {
 // FlagErrorFunc prints an error message which matches the format of the
 // docker/docker/cli error messages
 func FlagErrorFunc(cmd *cobra.Command, err error) error {
-	if err == nil {
+	if err != nil {
```

Comment:
This is not right; this function needs to print an error in the right format if it's not nil, and otherwise return without doing anything

It could've used `return nil` instead; perhaps it's _slightly_ clearer, but from a code PoV that won't make a difference